### PR TITLE
Add strong names

### LINKED ISSUE #407: Setup strong keys

### PR DESCRIPTION
**Pull request by kblok**
closes #407

### CODE REVIEW

File: src/PlaywrightSharp.Abstractions/PlaywrightSharp.Abstractions.csproj [hunk 1]
```diff
@@ -12,6 +12,9 @@
         <StyleCopTreatErrorsAsWarnings>false</StyleCopTreatErrorsAsWarnings>
         <CodeAnalysisRuleSet>../PlaywrightSharp.ruleset</CodeAnalysisRuleSet>
         <LangVersion>8.0</LangVersion>
+        <SignAssembly>true</SignAssembly>
```
Comment (Meir017) [inline src/PlaywrightSharp.Abstractions/PlaywrightSharp.Abstractions.csproj:15]: is it possible to have these tags in the common props files?

File: src/PlaywrightSharp.Chromium/PlaywrightSharp.Chromium.csproj [hunk 2]
```diff
@@ -1,30 +1,31 @@
...
+    </PropertyGroup>
+    <ItemGroup>
+        <ProjectReference Include="..\PlaywrightSharp\PlaywrightSharp.csproj" />
+        <ProjectReference Include="..\PlaywrightSharp.Abstractions\PlaywrightSharp.Abstractions.csproj" />
+    </ItemGroup>
+    <Import Project="../Common/Dependencies.props" />
+    <Import Project="../Common/SignAssembly.props" />
+    <ItemGroup>
+        <AssemblyAttribute Include="System.Runtime.CompilerServices.InternalsVisibleTo">
+            <_Parameter1>PlaywrightSharp.Tests, PublicKey=0024000004800000940000000602000000240000525341310004000001000100059a04ca5ca77c9b4eb2addd1afe3f8464b20ee6aefe73b8c23c0e6ca278d1a378b33382e7e18d4aa8300dd22d81f146e528d88368f73a288e5b8157da9710fe6f9fa9911fb786193f983408c5ebae0b1ba5d1d00111af2816f5db55871db03d7536f4a7a6c5152d630c1e1886b1a0fb68ba5e7f64a7f24ac372090889be2ffb</_Parameter1>
```
Comment (Meir017) [inline src/PlaywrightSharp.Chromium/PlaywrightSharp.Chromium.csproj:24]: can we have the `PublicKey` as a constant somewhere

Comment (kblok) [reply src/PlaywrightSharp.Chromium/PlaywrightSharp.Chromium.csproj]: I love these push backs, thank you!

**Comment by DrNeil** [comment]
getting latest DLL version with nuget, and sn.exe fails to __verify__ the strong name 
Is the key corrupt or some other issue ?
```
C:\Work\Spikes\test\bin\Debug\net5.0>sn.exe  -v PlaywrightSharp.dll

Microsoft (R) .NET Framework Strong Name Utility  Version 4.0.30319.0
Copyright (c) Microsoft Corporation.  All rights reserved.

Failed to verify assembly -- Strong name validation failed.
```

**Pull request merged**